HousingSearchApi.Tests/V1/E2ETests/Steps/GetProcessesSteps.cs

```diff
@@ -26,29 +26,46 @@ public async Task WhenRequestDoesNotContainSearchString()
 
         public async Task WhenRequestContainsSearchString()
         {
-            _lastResponse = await _httpClient.GetAsync(new Uri("api/v1/search/processes?searchText=%20abc", UriKind.Relative)).ConfigureAwait(false);
+            _lastResponse = await _httpClient.GetAsync(new Uri($"api/v1/search/processes?searchText={ProcessFixture.Processes[0].PatchAssignment.PatchId}", UriKind.Relative)).ConfigureAwait(false);
```

Comment:
Why don't you make _patchAssiigment public and then just reference it here rather than getting it from the Processes List?